storage/storage_test.go

```diff
@@ -590,10 +590,10 @@ func TestWritePartition(t *testing.T) {
 			}
 		}()
 		bd.Name = path.Base(file)
-		part1 := &BlockDevice{Name: bd.Name + "p1", FsType: "vfat", Size: 157286400, PartitionLabel: "CLR_BOOT", Type: BlockDeviceTypePart, MountPoint: "/boot"}
-		part2 := &BlockDevice{Name: bd.Name + "p2", FsType: "swap", Size: 125829120, PartitionLabel: "CLR_SWAP", Type: BlockDeviceTypePart, MountPoint: ""}
-		part3 := &BlockDevice{Name: bd.Name + "p3", FsType: "ext4", Size: 502267904, PartitionLabel: "CLR_ROOT_F_E", Type: BlockDeviceTypePart, MountPoint: "/"}
-		part4 := &BlockDevice{Name: bd.Name + "p4", FsType: "ext4", Size: 502267904, PartitionLabel: "CLR_MNT_/home", Type: BlockDeviceTypeCrypt, MountPoint: "/home"}
+		part1 := &BlockDevice{Name: bd.Name + "p1", FsType: "vfat", Size: 157286400, PartitionLabel: "CLR_BOOT", Type: BlockDeviceTypePart, MountPoint: "/boot", MakePartition: true}
+		part2 := &BlockDevice{Name: bd.Name + "p2", FsType: "swap", Size: 125829120, PartitionLabel: "CLR_SWAP", Type: BlockDeviceTypePart, MountPoint: "", MakePartition: true}
+		part3 := &BlockDevice{Name: bd.Name + "p3", FsType: "ext4", Size: 502267904, PartitionLabel: "CLR_ROOT_F_E", Type: BlockDeviceTypePart, MountPoint: "/", MakePartition: true}
```

Comment:
We should drop the _E here as we do not currently allow encryption during advanced.
```suggestion
		part3 := &BlockDevice{Name: bd.Name + "p3", FsType: "ext4", Size: 502267904, PartitionLabel: "CLR_ROOT_F", Type: BlockDeviceTypePart, MountPoint: "/", MakePartition: true}
```

Reply:
Done